src/dashboard/testUtils/index.js

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from 'react';
+import { ApolloProvider } from 'react-apollo-hooks';
+import { render } from '@testing-library/react';
+
+import { createApolloMockClient } from './apolloMockClient';
+
+export const renderWithContext = (component, { mocks }) => {
+    const mockApolloClient = createApolloMockClient(mocks);
+
+    // Warning: don't call any functions in here that return new instances on every call.
+    // All the props need to stay the same otherwise render won't work.
```

Comment:
Can you make this say:
```suggestion
    // All the props need to stay the same otherwise rerender won't work.
```
I wrote it wrong... I was having issues of the props on the provider changing and rerendering the entire component tree, not just the component under test.